### ISSUE TITLE
React Examples suggestion

### ISSUE THREAD

**Issue by TheFern2** [tauri-apps/examples#542]
I think the examples in general can gain from some additional details to show what is possible, but since I am using react I put that on the title. The example is really just cra cli + tauri cli added and init with nothing else added, that's something anyone can do in the cli. Would be useful if you show how to call rust from a js file, how to call a promise, how to call a rust function from js that returns something, etc. And anything else that's possible with tauri that's not possible with electron so people can see the contrast. Then people can decide if is worth using tauri or not, for me coming from JS I want to be able to see all the possibilities and if I like what I see in terms of performance and binary size then I can spend some time to learn rust properly. 

- To summarize show the IPC in the examples.

I clicked on every tauri react example on github, and 99% are really just templates for cra + tauri. I think only one had an example of using the Promisified function. There isn't really much out there as far as tutorials or examples, so it makes it really hard to do anything with it.

**Comment by lucasfernog** [comment]
We're moving the showcase examples to the tauri monorepo.
